docs/notifications.json

```diff
@@ -1,10 +1,15 @@
 [
   {
     "id": 27,
-    "text": "You can <a style=\"color: white;\" target=\"_blank\" rel=\"noopener\" href=\"https://twitter.com/MaterialUI\">follow us on Twitter</a> to receive exclusive tips and updates about Material-UI and React ecosystem."
+    "text": "You can <a style=\"color: white;\" target=\"_blank\" rel=\"noopener\" href=\"https://twitter.com/MaterialUI\">follow us on Twitter</a> to receive exclusive tips and updates about Material-UI and the React ecosystem."
   },
   {
     "id": 33,
-    "text": "Material-UI v3.9.0 is out 🎉. Please see <a style=\"color: white;\" target=\"_blank\" rel=\"noopener\" href=\"https://github.com/mui-org/material-ui/releases/tag/v3.9.0\">the release notes</a>."
+    "text": "Material-UI v3.9.0 is out 🎉. Please check <a style=\"color: white;\" target=\"_blank\" rel=\"noopener\" href=\"https://github.com/mui-org/material-ui/releases/tag/v3.9.0\">the release notes</a> for more details."
+  },
+  {
+    "id": 34,
+    "text": "Lets translate! Ajude a comunidade a <a style=\"color: white;\" target=\"_blank\" rel=\"noopener\" href=\"https://translate.material-ui.com/\">traduzir a documentação</a> para o Português. 🇧🇷"
```

Comment:
Can we translate the edit button?

Reply:
Already done - I was just about to update the PR. 😄 

(I was originally going to submit it separately but decided to combine it with this one.)